config/environment.d.ts

```diff
@@ -120,6 +120,7 @@ declare const config: {
             support: string;
             dashboard: string;
             home: string;
+            institutions: string;
```

Comment:
`config/environment.d.ts` has been updated such that you don't need to add each flag anymore (you'll notice a merge conflict).